cyral/resource_cyral_repository_test.go

```diff
@@ -60,5 +62,6 @@ func formatRepoDataIntoConfig(data RepoData) string {
 		host  = "%s"
 		port  = %d
 		name  = "%s"
-	}`, data.RepoType, data.Host, data.Port, data.Name)
+		labels = ["%s"]
+	}`, data.RepoType, data.Host, data.Port, data.Name, data.Labels)
```

Comment:
Notice that this will procude the followin labels:
```
labels = ["[rds us-east-1]"]
```
instead of 
```
labels = ["rds", "us-east-1"]
```
There's a util function named [formatAttibutes](https://github.com/cyralinc/terraform-provider-cyral/blob/main/cyral/resource_cyral_policy_test.go#L80) that I think its used to split the labels properly.

Reply:
Thanks @VictorGFM, this function is exactly what I needed. I tested here and now the output is exactly what I expect